.golangci.yml

```diff
@@ -45,12 +45,12 @@ linters:
     - errorlint # finds code that will cause problems with the error wrapping scheme introduced in Go 1.13
     - exhaustive # checks exhaustiveness of enum switch statements
     - exptostd # detects functions from golang.org/x/exp/ that can be replaced by std functions
-    - fatcontext # detects nested contexts in loops
+    #- fatcontext # detects nested contexts in loops
```

Comment:
We shouldn't disable any of these checks. 
These are actually helpful. This is a public project and keep the standards high from the beginning is important.

If you have exceptions for that you need to apply for the code is not mission critical like examples just use inline directives.

Reply:
Let me fix the lint in this PR